redbot/core/bank.py

```diff
@@ -215,7 +215,7 @@ def _invalid_amount(amount: int) -> bool:
         raise ValueError("Invalid withdrawal amount {} <= 0".format(amount))
 
     bal = await get_balance(member)
-    if amount > bal:
+    if amount >= bal:
```

Comment:
This one was right originally, `withdraw_credits` shouldn't raise if the amount is equal to the balance.

Reply:
I'm confused as to what you are talking about. withdraw_credits will raise if the amount is equal to the balance.
Both can_spend and withdraw_credits are using greater than instead of greater than or equal to. I can write a mock cog for you to test both functions if you like. I'm unsure if you are in agreement or not lol.

Reply:
Nvm I was looking at that particular spot wrong. Sorry about that.